R/SSexecutivesummary.r

```diff
@@ -270,12 +283,12 @@ SSexecutivesummary <- function(replist,
       } else {
         es.a <- data.frame(years_minus_final, catch, total.catch, total.dead)
       }
-      colnames(es.a) <- c("Year", paste(fleet.names, "(mt)"), "Total Catch (mt)", "Total Dead (mt)")
+      colnames(es.a) <- c("Year", paste(fleet.names, "(mt)"), "Total Landings (mt)", "Total Catch (mt)")
```

Comment:
Thanks Ian for the suggestion. I never thought about discards that survive. I was only thinking that total dead did not include natural mortality so it really isn't the total number dead. I will change it back to "Total Dead (mt)" under the assumption that people will know that the table is referring to dead catches because it is a table of catch histories :wink:.

Reply:
We could also use "Total Dead Catch (mt)" but that seems too long.

Reply:
🤣 I thought the same thing; but, I did notice that table f uses "Total Mortality" so maybe we should just use that to make it consistent?

Reply:
I think there has been some discussion that total mortality should also include natural mortality. I think using Total Dead is likely sufficient in this instance since it is fairly clear what it is referring to.